chainer/function.py

```diff
@@ -174,6 +174,16 @@ def __call__(self, *inputs):
             return ret[0]
         return ret
 
+    @property
+    def label(self):
+        """Default implementation, which returns function's property
```

Comment:
This is not a summary, some "additional" information. Write what the property "label" means, which should give enough information when one wants to override it. For example:
```
A short text that represents this function.

The default implementation just returns the type name.
Each function implementation should override it to give more information.
```